### DISCUSSION TITLE
Hello teachers! After updating, an error was reported. How should this error be resolved? Please advise!

### DISCUSSION THREAD

**Discussion by zjgzh001** [trickerer/Trinity-Bots#479] [General]
Hello teachers! After updating, an error was reported. How should this error be resolved? Please advise!
![image](https://github.com/trickerer/Trinity-Bots/assets/133203282/38f36a93-49b0-447f-8d6a-0ad42d8cb969)

**Comment by trickerer** [comment]
You have new SQLs applied to your DB but you run an outdated version of your server executable.
Re-build and make sure you use a new one.

**Reply by zjgzh001** [reply]
Thank you for your guidance. I have fixed this error according to your prompt. Thank you very much!
